workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/GameObjectDispatcherSystem.cs

```diff
@@ -22,7 +22,7 @@ namespace Improbable.Gdk.GameObjectRepresentation
     [DisableAutoCreation]
     [AlwaysUpdateSystem]
     [UpdateInGroup(typeof(SpatialOSReceiveGroup.GameObjectReceiveGroup))]
-    internal class GameObjectDispatcherSystem : ComponentSystem
+    public class GameObjectDispatcherSystem : ComponentSystem
```

Comment:
If this is only used in tests that I'd rather make it visible to the test dll than make it public.

Reply:
Alternatively we should use the helper methods to add these systems to test worlds.

Reply:
Will do